katas/content/oracles/index.md

```diff
@@ -359,6 +357,6 @@ However, if you're designing an oracle for a new problem, you do not have a refe
 
 A good way to test a quantum oracle of interest is to write a classical oracle that performs the same computation classically, and then compare the effect of your quantum oracle on the basis states with the output of the classical oracle for every input (or a lot of the inputs if you are constrained by runtime) to ensure that they match.
 
-Here we will test your implementation from task 4.3 by comparing it to the classical code implementing the same function. 
+Here we will compare reference implementation of `Meeting_Classical` to the classical code implementing the same function. 
```

Comment:
Again, seems like it should read "we will compare a reference implmentation", or "the reference implementation".